index.html

```diff
@@ -17,8 +17,42 @@
     <!-- now loading css -->
     <link href="css/loading.css" rel="stylesheet" type="text/css" />
     <link href="css/style.css" rel="stylesheet" type="text/css" />
+    <style> .announcement-banner { 
+      position: fixed; 
+      top: 0; left: 0; 
+      width: 100%; 
+      background-color: #f8f8f8; 
+      padding: 10px; 
+      text-align: center; 
+      z-index: 9999; 
+      } 
+      .announcement-banner .close-button { 
+        float: right; 
+        cursor: pointer; 
+      } 
+    </style> 
```

Comment:
Same with index.html, should this go in style.css since its being used in 2 files?

Reply:
Sure